lib/core/styles/_a11y.scss

```diff
@@ -0,0 +1,8 @@
+.adf-sr-only {
```

Comment:
better to add in card-view-dateitem.component.scss

Reply:
this can be used globally, not just in cards. anything you want to make available only to screenreaders can use this class. is this still an appropriate location?